src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -2120,7 +2150,7 @@ void AutomationInstrumentClipView::modEncoderAction(int32_t whichModEncoder, int
 	ModelStack* modelStack = setupModelStackWithSong(modelStackMemory, currentSong);
 
```

Comment:
updated this UI mode check because it wasn't working if another UI (e.g. auditioning) was active at the time as a note is pressed